System/Remote/Monitoring.hs

```diff
@@ -199,13 +201,23 @@ data Server = Server {
 -- | Like 'forkServerWith', but creates a default metric store with
 -- some predefined metrics. The predefined metrics are those given in
 -- 'System.Metrics.registerGcMetrics'.
-forkServer :: Maybe S.ByteString  -- ^ Host to listen on (e.g. \"localhost\")
+forkServer :: S.ByteString  -- ^ Host to listen on (e.g. \"localhost\")
            -> Int           -- ^ Port to listen on (e.g. 8000)
            -> IO Server
 forkServer host port = do
     store <- Metrics.newStore
     Metrics.registerGcMetrics store
-    forkServerWith store host port
+    forkServerMaybeHostnameWith store (Just host) port
+
+-- | If you are running EKG on a private network (including virtual
+-- private network), it may be appropriate to bind to all interfaces,
+-- not only localhost.
+forkServerNoHostname :: Int           -- ^ Port to listen on (e.g. 8000)
+           -> IO Server
```

Comment:
Please format type signatures in the style used in this file (i.e. arrows lined up with `::`).